postgres_insert_block.py

```diff
@@ -0,0 +1,62 @@
+from nio.properties import BoolProperty, StringProperty
+from nio.util.discovery import discoverable
+from .postgres_base_block import PostgresBase
+
+
+@discoverable
```

Comment:
FYI, this is technically not required anymore. @hansmosh what are we saying is our standard way of decorating blocks?